src/store/store.ts

```diff
@@ -0,0 +1,1218 @@
+          isDocumentDeliveryValid &&
+            isTransactionalFnValid &&
+            isCertifyValid &&
+            isCourtOrderValid &&
+            isStaffPaymentValid &&
+            isDissolutionDateValid &&
+            isCompletingPartyValid
+        )
+      }
+      return (
+        this.isDissolutionDefineDissolutionValid &&
+        this.isAffidavitValid &&
+        this.isResolutionValid &&
+        isDocumentDeliveryValid &&
+        isTransactionalFnValid &&
+        isCertifyValid &&
+        isEffectiveDateTimeValid &&
+        isCourtOrderValid &&
+        isStaffPaymentValid
+      )
+    },
+
+    /** Whether all the incorporation steps are valid. */
+    isIncorporationApplicationValid (): boolean {
+    // Base company steps
+      const isBaseStepsValid = (
+        this.getCreateShareStructureStep.valid &&
+        this.getEffectiveDateTime.valid &&
+        this.getIncorporationAgreementStep.valid &&
+        this.getCourtOrderStep.valid
+      )
+
+      // Coop steps
+      const isCoopStepsValid = (
+        this.getCooperativeType &&
+        this.getCreateRulesStep.validationDetail.valid &&
+        this.getCreateMemorandumStep.validationDetail.valid
+      )
+
+      // Validate different steps for Base Companies vs Coops
+      const isDocumentValid = this.isBaseCompany ? isBaseStepsValid : isCoopStepsValid
+
+      const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
+
+      return (
+        this.isDefineCompanyValid &&
+        this.isAddPeopleAndRolesValid &&
+        isDocumentValid &&
+        isCertifyValid
+      )
+    },
+
+    /** Whether all the registration steps are valid. */
+    isRegistrationValid (): boolean {
+      const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
+      // const isFeeAcknowledgementValid = getRegistration.feeAcknowledgement
+      const isFeeAcknowledgementValid = true // FUTURE: use line above instead
+      const isStaffPaymentValid = this.isRoleStaff ? this.getStaffPaymentStep.valid : true
+
+      return (
+        this.getRegistration.defineBusinessValid &&
+        this.isAddPeopleAndRolesValid &&
+        isCertifyValid &&
+        isFeeAcknowledgementValid &&
+        isStaffPaymentValid
+      )
+    },
+
+    /** Whether all the restoration steps are valid. */
+    isRestorationValid (): boolean {
+      const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
+      const isStaffPaymentValid = this.isRoleStaff ? this.getStaffPaymentStep.valid : true
+
+      return (
+        this.isRestoreBusinessNameValid && // step 1
+        this.isAddPeopleAndRolesValid && // step 2
+        this.isDefineCompanyValid && // step 3
+        isCertifyValid && // step 4
+        isStaffPaymentValid // step 4
+      )
+    },
+
+    /**
+   * Is true when the user has tried to submit a filing,
+   * ie, after clicking File and Pay.
+   */
+    getValidateSteps (): boolean {
```

Comment:
Indentation. Should be:
```
    /**
     * Is true when the user has tried to submit a filing,
     * ie, after clicking File and Pay.
     */
    getValidateSteps (): boolean {
```

Check other multi-line comments below.

Also check if `npm run lint` auto-fixes these.

Reply:
Looks good, thanks.